### PR TITLE
feat: replace errors.ErrorStack to terror.Message

### LINKED ISSUE #502: UCP: remove the call stack in error msg of `query-status`
## Feature Request

**Is your feature request related to a problem? Please describe:**

dm

the error msg including the call stack is ugly for the user, and  might even make the user ignore it.

**Describe the feature you'd like:**

remove the call stack in error msg, like use `terror.Message(err)` to replace `errors.ErrorStack(err)` in `NewProcessError`.

## Score

50

## Mentor(s)

* @csuzhangxc 

Contact the mentors: **#tidb-challenge-program** channel in [TiDB Community](https://join.slack.com/t/tidbcommunity/shared_invite/enQtNzc0MzI4ODExMDc4LWYwYmIzMjZkYzJiNDUxMmZlN2FiMGJkZjAyMzQ5NGU0NGY0NzI3NTYwMjAyNGQ1N2I2ZjAxNzc1OGUwYWM0NzE)  Slack Workspace

## Recommended Skills

* Golang
* Experience with DM

## Learning Materials

- [DM source reading](https://pingcap.com/blog-cn/#DM-%E6%BA%90%E7%A0%81%E9%98%85%E8%AF%BB)
- [DM document](https://pingcap.com/docs-cn/stable/reference/tools/data-migration/overview/#dm-%e6%9e%b6%e6%9e%84)

### PR DESCRIPTION
**Pull request by Kuri-su**
UCP #502 

### What problem does this PR solve? 

use terror.Message(err) to replace errors.ErrorStack(err) in 
- NewProcessError  
- dm/ctl/master/query_error.go
- dm/ctl/master/query_status.go

### What is changed and how it works?

fix #502

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@lichunzhu PTAL

File: dm/unit/unit.go [hunk 1]
```diff
@@ -65,7 +63,7 @@ type Unit interface {
 // we can refine to add error scope field if needed
 func NewProcessError(err error) *pb.ProcessError {
 	result := &pb.ProcessError{
-		Msg: errors.ErrorStack(err),
+		Msg: terror.Message(err),
```
Comment (lichunzhu) [inline dm/unit/unit.go]:
```suggestion
		Msg: "",
```

Comment (lichunzhu) [reply dm/unit/unit.go]: After a discussion we think it's better to set empty Msg here and return `result.Error` to users.
But we set `result.Error` to nil in https://github.com/pingcap/dm/blob/master/dm/worker/status.go#L206. Could you help us to remove this line?
What's more, there are some unit tests and code (for example: https://github.com/pingcap/dm/blob/master/checker/cmd.go#L73) using `result.Msg`, can you help us refine these code?

Comment (Kuri-su) [reply dm/unit/unit.go]: ok, let me see

Comment (lichunzhu) [inline dm/unit/unit.go]: Can we set `Msg` to "" ? It's weired that users can get the similar message from `Msg` and `Error`.

Comment (Kuri-su) [reply dm/unit/unit.go]: wait:joy:, In fact, I just changed it, but I still can't pass the test, and then rolled back some to verify which error was caused. Finally, it seems that it is necessary to modify the test case. :joy:

Comment (Kuri-su) [reply dm/unit/unit.go]: And i was just deploying a test environment on my PC :joy:

File: dm/unit/unit.go [hunk 2]
```diff
@@ -65,7 +63,7 @@ type Unit interface {
 // we can refine to add error scope field if needed
 func NewProcessError(err error) *pb.ProcessError {
 	result := &pb.ProcessError{
-		Msg: errors.ErrorStack(err),
+		Msg: "",
 	}
 	if e, ok := err.(*terror.Error); ok {
```
Comment (lichunzhu) [inline dm/unit/unit.go]: When `ok` is false, we should set `result.Msg = err.Error()`

Comment (lichunzhu) [reply dm/unit/unit.go]: Remove errorStack in https://github.com/pingcap/dm/blob/master/dm/master/server.go#L1521

Comment (Kuri-su) [reply dm/unit/unit.go]: @lichunzhu done

**Comment by lichunzhu** [comment]
Could you please take a look at slack? I have left a message for you.

**Comment by Kuri-su** [comment]
@lichunzhu Done

**Pull request merged**